src/overlay.jsx

```diff
@@ -133,7 +95,9 @@ const Overlay = React.createClass({
     const styles = getStyles(this.props, this.state);
 
     return (
-      <div {...other} ref="overlay" style={prepareStyles(Object.assign(styles.root, style))} />
+      <div {...other} ref="overlay" style={prepareStyles(Object.assign(styles.root, style))}>
+        {this.props.autoLockScrolling && <AutoLockScrolling lock={this.props.show} />}
```

Comment:
You are right, this component is only used by `Overlay` and `LeftNav`.
The `autoLockScrolling` is not used. I think that we should remove it :+1:.